### DISCUSSION TITLE
github copilot when gpt 4?

### DISCUSSION THREAD

**Discussion by zaelii** [community/community#50654] [Copilot Conversations]
does anyone know if github copilot will use gpt 4 ?
is it gonna make any diference?

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @zaelii , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
